frame/bounties/src/weights.rs

```diff
@@ -126,62 +151,87 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	// Storage: Treasury BountyApprovals (r:1 w:1)
 	fn approve_bounty() -> Weight {
-		(12_417_000 as Weight)
+		(14_042_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
 	}
+	// Storage: Treasury Bounties (r:1 w:1)
 	fn propose_curator() -> Weight {
-		(9_692_000 as Weight)
+		(11_059_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
+	// Storage: Treasury Bounties (r:1 w:1)
+	// Storage: System Account (r:1 w:1)
 	fn unassign_curator() -> Weight {
-		(41_211_000 as Weight)
+		(44_858_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
 	}
+	// Storage: System Account (r:1 w:1)
+	// Storage: Treasury Bounties (r:1 w:1)
 	fn accept_curator() -> Weight {
-		(37_376_000 as Weight)
+		(40_789_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
 	}
+	// Storage: Treasury Bounties (r:1 w:1)
 	fn award_bounty() -> Weight {
-		(25_525_000 as Weight)
+		(27_810_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
+	// Storage: System Account (r:3 w:3)
+	// Storage: Treasury BountyDescriptions (r:0 w:1)
+	// Storage: Treasury Bounties (r:1 w:1)
 	fn claim_bounty() -> Weight {
-		(125_495_000 as Weight)
+		(134_983_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(5 as Weight))
 	}
+	// Storage: Treasury Bounties (r:1 w:1)
+	// Storage: System Account (r:1 w:1)
+	// Storage: Treasury BountyDescriptions (r:0 w:1)
 	fn close_bounty_proposed() -> Weight {
-		(40_464_000 as Weight)
+		(44_256_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(3 as Weight))
 	}
+	// Storage: System Account (r:2 w:2)
+	// Storage: Treasury BountyDescriptions (r:0 w:1)
+	// Storage: Treasury Bounties (r:1 w:1)
 	fn close_bounty_active() -> Weight {
-		(84_042_000 as Weight)
+		(90_401_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(3 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
 	}
+	// Storage: Treasury Bounties (r:1 w:1)
 	fn extend_bounty_expiry() -> Weight {
-		(25_114_000 as Weight)
+		(27_036_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
+	// Storage: System Account (r:2 w:2)
+	// Storage: Treasury Bounties (r:1 w:1)
+	// Storage: Treasury BountyApprovals (r:1 w:1)
 	fn spend_funds(b: u32, ) -> Weight {
-		(351_000 as Weight)
-			// Standard Error: 13_000
-			.saturating_add((58_724_000 as Weight).saturating_mul(b as Weight))
+		(13_740_000 as Weight)
```

Comment:
big change here

Reply:
Maybe related to the decoding overhead added by gui

Reply:
I didn't update the benchmark I thought you said you would do, though I had run some of them in this PR https://github.com/paritytech/substrate/pull/9413

Reply:
on the linked PR there is actually a base weight of 0 :-|